developers/weaviate/search/multi-vector.md

```diff
@@ -98,6 +101,57 @@ Specify query vectors as a dictionary/map of names and vectors.
 
 You can also specify the query vectors as an array of vectors. The array will be parsed according to the order of the specified target vectors.
 
+### Specify array(s) of query vectors
+
+:::info Added in `v1.27`
+:::
+
+You can specify the value of the query vector dictionary/map as an array of vectors.
+
+#### Target vectors as an array
+
+The target vectors can be specified as an array as shown here.
+
+<Tabs groupId="languages">
+<TabItem value="py" label="Python Client v4">
+<FilteredTextBlock
+  text={PyCodeV4}
+  startMarker="# START MultiTargetMultipleNearVectorsV1"
+  endMarker="# END MultiTargetMultipleNearVectorsV1"
+  language="python"
+/>
+</TabItem>
+<TabItem value="ts" label="JS/TS Client v3">
+
+```ts
+// Coming soon
+```
+
+</TabItem>
+</Tabs>
+
+#### Target vectors as a map of weights
+
+The target vectors can be specified as a dictionary/map of weights and vectors as shown below.
```

Comment:
I also find this a bit unclear.

How about something like:
"If you want to provide weights for each target vector you can do it like so"

Reply:
Also edited :)